packages/compiler-ssr/src/transforms/ssrTransformElement.ts

```diff
@@ -87,6 +87,17 @@ export const ssrTransformElement: NodeTransform = (node, context) => {
     const hasCustomDir = node.props.some(
       p => p.type === NodeTypes.DIRECTIVE && !isBuiltInDirective(p.name),
     )
+
+    // v-show has a higher priority in ssr
+    const vShowPropIndex = node.props.findIndex(
```

Comment:
I think this should be processed in [injectFallthroughAttrs](https://github.com/vuejs/core/blob/bfe6b459d3a0ce6168611ee1ac7e6e789709df9d/packages/compiler-ssr/src/transforms/ssrInjectFallthroughAttrs.ts#L82). If props have v-show, place the `_attrs` before v-show.